packages/cozy-harvest-lib/src/datacards/FileDataCard.jsx

```diff
@@ -212,13 +220,19 @@ const FileDataCard = ({ filesCol, konnector, accountId }) => {
   const noFiles = hasQueryBeenLoaded(filesCol) && files.length == 0
   const isLoading = isQueryLoading(filesCol)
   return noFiles ? null : (
-    <FileCard files={files} loading={isLoading} konnector={konnector} />
+    <FileCard
+      files={files.slice(0, 5)}
```

Comment:
Wondering if we have this issue using RealTimequery

Reply:
Yep we still have the issue, queries inside cozy-client do not respect the limitBy parameter of the query.

Reply:
Issue created https://github.com/cozy/cozy-client/issues/901